en/components/combo.md

```diff
@@ -6,105 +6,88 @@ _keywords: Design Systems, Design Systems UX, UI kit, Figma, Figma to Angular, E
 
 # Combo
 
-Use the Combo component to allow users to select an option from a predefined list of choices or enter their own value. When the user clicks on the drop-down arrow, a drop down list of options is displayed, and they can either select one of the options or type their own value into the text box. The Combo is visually identical to the [ Ignite UI for Angular Combobox Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/combo.html)
+Use the Combo component to allow users to select an option from a predefined list of choices or enter their own value. When the user clicks on the arrow down icon, a Dropdown list of options is displayed, and they can select one of the options, search for an value, or type their own value into the text box. The Combo is visually identical to the [ Ignite UI for Angular Combobox Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/combo.html)
+
+> [!Note]
+> The Combo has two layout options: Combo with Multiple Select `(Combo)` and [Single Select Combo (Simple Combo) ](simple-combo.md)
 
 ## Combo Demo
 
-The Combo has two pieces: an input containing the current selection and a drop down that is shown in order to select one or more items.
+The Combo has two pieces: an input containing the current selection and a Dropdown that is shown in order to select one or more items.
```

Comment:
The Combo has two pieces: an input containing the current selection **or a placeholder** and a Dropdown, shown in order to select one or more items.

Reply:
Fixed